core/services/chainlink/config_general.go

```diff
@@ -60,7 +62,18 @@ func NewGeneralConfig(tomlString string) (coreconfig.GeneralConfig, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &generalConfig{c: &c, inputTOML: input, effectiveTOML: effective}, nil
+
+	// Secrets config does not have default values
```

Comment:
No defaults, but how/when are individual environment variables incorporated? Maybe they should be preset or override using the same sort of mechanism as the defaults?

Reply:
OK, I'll add support for ENV/flag overrides.